src/test/beast_Zero_test.cpp

```diff
@@ -31,7 +31,7 @@ int signum (adl_tester)
 }
 
 
-namespace detail {
+namespace inner_adl_test {
```

Comment:
Having the `signum` for `adl_tester` inside beast::detail was resulted in shadowing of the non-namespaced, generic `signum` in [Zero.h](https://github.com/ripple/rippled/blob/develop/src/ripple/beast/utility/Zero.h#L58)